bitnami/suitecrm/templates/deployment.yaml

```diff
@@ -187,11 +187,11 @@ spec:
               value: {{ .Values.suitecrmNotifyAddress | quote }}
             - name: SUITECRM_SMTP_HOST
               value: {{ .Values.suitecrmSmtpHost | quote }}
-            - name: SUITECRM_SMTP_PORT_NUMBER
+            - name: SUITECRM_SMTP_PORT
```

Comment:
The env. variable used to configure SMTP is actually `SUITECRM_SMTP_PORT_NUMBER`, see:

- https://github.com/bitnami/bitnami-docker-suitecrm/blob/master/7/debian-10/rootfs/opt/bitnami/scripts/libsuitecrm.sh#L303

That said, both are aliases so it would work using both formats. In any case, please revert this change

Reply:
Sorry, but when I open the link I only see SUITECRM_SMTP_PORT. Am I missing something?

Reply:
Sorry, I shared the wrong link... Here we're using `SUITECRM_SMTP_PORT_NUMBER`:
- https://github.com/bitnami/bitnami-docker-suitecrm/blob/master/7/debian-10/rootfs/opt/bitnami/scripts/libsuitecrm.sh#L74
It seems there's some inconsistency around this, I'll fix it in the container image. 
Here you can find they're actually aliases:
- https://github.com/bitnami/bitnami-docker-suitecrm/blob/master/7/debian-10/rootfs/opt/bitnami/scripts/suitecrm-env.sh#L86
But we should use `SUITECRM_SMTP_PORT_NUMBER` for consistency with other Bitnami containers.

Reply:
Thanks for the clarification. I Reverted the change. Thank you again.